### ISSUE TITLE
PaginationV2 arrow icon misalignment

### ISSUE THREAD

**Issue by tozhangkai** [IBM/carbon-components-react#2128]
## Detailed description

> The arrow icon after the page number in PaginationV2 doesn't align correctly. I believe this is a regression introduced recently. I didn't see it in the previous versions. See here:
http://react.carbondesignsystem.com/?selectedKind=Pagination&selectedStory=v2&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel
![image](https://user-images.githubusercontent.com/11453713/55261690-0105fe80-5242-11e9-8c0f-e61760286dc7.png)

> Is this a feature request (new component, new icon), a bug, or a general issue? 
This is a bug

> Is this issue related to a specific component?
Yes, `PaginationV2` in `carbon-components-react`

> What did you expect to happen? What happened instead? What would you like to see changed?
I expected the icons align to the vertical center of the component.

> What browser are you working in?
Firefox, Chrome and Safari. The problem happens on all of them and doesn't seem to be related to a specific version of the browsers.

> What version of the Carbon Design System are you using?
`carbon-components-react:  6.114.0`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
I am working on an IBM product that has a quarterly release cycle, and we're moving to React with extensive use of carbon components.  

## Steps to reproduce the issue
The problem can be reproduced in the carbon-components online storybook:
http://react.carbondesignsystem.com/?selectedKind=Pagination&selectedStory=v2&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

## Additional information
See the screenshot above in the description section

**Comment by asmith-IBM** [comment]
I'm particularly interested in this fix as it's blocking me updating to the latest version of v6. I had a brief look into this and have a feeling the regression was introduced in #2093 as it's the only major styling update I can see since this component worked.

**Comment by dakahn** [comment]
Looking at your codepen I'm not seeing this misalignment. Maybe this was fixed @tozhangkai ?

**Comment by asmith-IBM** [comment]
Hi, I believe this issue refers to v6 and as such the link in the issue is not accurate. You can clearly see the arrow issue at: http://v6-react.carbondesignsystem.com/?selectedKind=Pagination&selectedStory=v2&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

I am eagerly awaiting a fix for this issue also. Had a brief look into the cause but didn't really get anywhere. Anything I can do to help please let me know.

**Comment by asudoh** [comment]
@asmith-IBM Yeah contributions will be great - The style is hosted in `v9` branch of `carbon-components` repo.

**Comment by tozhangkai** [comment]
@asmith-IBM thanks for pointing out this is for carbon react v6 (or carbon v9). The link I provided used to point to the v6 code and now it appears to have been moved to the latest code base. @asudoh is there an estimation on when it'll be fixed? It becomes a show stopper for our product as we have pagination in many of the pages, and the problem is pretty obvious. The new v7 (or carbon v10) doesn't have the problem, but it's a totally new design and we can't afford to move to it recently. Thanks.

**Comment by asudoh** [comment]
@tozhangkai This issue is not marked as high priority as application can do (small) style override to address this. As I noted earlier, contributions will be nice - Thanks!

**Comment by wilhaddenIBM** [comment]
@asudoh could you post the style override is to fix this? Thanks.

**Comment by asudoh** [comment]
@wilhaddenIBM I don't have one I can rattle off quickly - You can use DOM inspector in Chrome DevTools to fiddle with the styles to see what change yields the style you desire. Once you figure it out, be sure to consider contributing back. Thanks!
